### DISCUSSION TITLE
Lost access to organization account

### DISCUSSION THREAD

**Discussion by cristianobrt** [community/community#74752] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, the company where I work has lost access to its organization account. How do I recover?

**Comment by gitstua** [comment]
Hi there! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by cristianobrt** [reply]
Hello @gitstua, i opened a support ticket and have not received a response. Anything else I can do?

**Reply by srvchawla** [reply]
@cristianobrt Have you tried recovering your account with the [recovery](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials) methods?
